lib/msal-core/src/utils/RequestUtils.ts

```diff
@@ -75,13 +78,40 @@ export class RequestUtils {
         return validatedRequest;
     }
 
+    /**
+     * 
+     * @param request 
+     * @param clientId 
+     * @param interactionType 
+     */
+    static validateLoginRequest(request: AuthenticationParameters, clientId: string, interactionType: InteractionType): AuthenticationParameters {
+        // Check if userRequest is empty
+        if (!request) {
+            request = {};
+        }
+
+        // Check if request scopes are null
+        if (!request.scopes) {
+            request.scopes = [];
+        }
+
+        // Append clientId to scopes by default for login calls
+        request.scopes = ScopeSet.appendScopes(request.scopes, [clientId]);
+
+        // validate request
+        const userRequest: AuthenticationParameters = RequestUtils.validateRequest(request, clientId, interactionType);
+        userRequest.scopes = ScopeSet.appendScopes(userRequest.scopes, userRequest.extraScopesToConsent);
```

Comment:
Echoing what @pkanher617 noted on the original branch:

This was being called before validateInputScopes before, is now being called afterwards - is that ok? Would have failed the client id check otherwise, but since that was removed might be better to move it before validateRequest.